l10n_it_declaration_of_intent/wizard/select_declarations.py

```diff
@@ -47,6 +50,7 @@ def confirm(self):
         if not invoice_id:
             return res
         invoice = self.env["account.move"].browse(invoice_id)
-        for declaration in self.declaration_ids:
-            invoice.declaration_of_intent_ids = [(4, declaration.id)]
+        invoice.declaration_of_intent_ids = [
+            fields.Command.set(self.declaration_ids.ids)
+        ]
```

Comment:
```suggestion
        invoice.declaration_of_intent_ids = self.declaration_ids
```
È un po' più semplice e credo sia equivalente, cosa ne pensi?